Website/about.html

```diff
@@ -0,0 +1,307 @@
+<!DOCTYPE html>
```

Comment:
Am incercat sa ma uit pe PR. Spun "am incercat" pentru ca mi-e imposibil sa fac review **de calitate** la asemenea cantitate de cod. :D De exemplu, fisierele cu peste 4.000 de linii de css (uneori chiar peste 10.000) imi depasesc puterile de concentrare. :)))

Template-ul gasit de tine arata foarte bine si cred ca poti sa te inspiri din el (si sa refolosesti) anumite parti dar nu in felul acesta. O sa devina extrem de greu de mentinut un asemenea UI avand in vedere ca, cel mai probabil, nici macar nu ai reusit sa vezi tot codul - de inteles tot codul nici nu se mai pune problema (inteles pe bune, care chiar sa te ajute in viitoarea ta cariera ca in mare sigur te-ai prins ce e acolo).

Concluzie: e un starting point foarte bun dar cred ca cel mai bine ar fi sa fie pastrat ca sursa de inspiratie si nu copy-paste complet.